### DISCUSSION TITLE
docker-compose up: Cannot start services superset_init，uperset_worker_beat。。。

### DISCUSSION THREAD

**Discussion by happy-hui1** [apache/superset#19950] [Q&A / Help]
#### How to reproduce the bug
Removing superset_app
Removing superset_init
Removing superset_worker
Removing superset_worker_beat
superset_cache is up-to-date
superset_db is up-to-date
Starting superset_websocket ... 
Recreating 4e80981fc5ad_superset_app ... 
Starting superset_node               ... 
Starting superset_websocket                  ... done
Recreating a10d3d77804a_superset_worker ... 
Starting superset_node                       ... done
Recreating bc04ec33c40f_superset_init        ... error

ERROR: for 6c79d9ae2119_superset_tests_worker  Cannot start service superset-tests-worker: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

Recreating cc76d3717563_superset_worker_beat ... errorocess caused: exec: "/app/docker/docker-init.sh": permission denied: unknown
Recreating 4e80981fc5ad_superset_app         ... error
ERROR: for cc76d3717563_superset_worker_beat  Cannot start service superset-worker-beat: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

ERROR: for a10d3d77804a_superset_worker  Cannot start service superset-worker: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

ERROR: for 4e80981fc5ad_superset_app  Cannot start service superset: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

ERROR: for superset-tests-worker  Cannot start service superset-tests-worker: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

ERROR: for superset-init  Cannot start service superset-init: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-init.sh": permission denied: unknown

ERROR: for superset-worker-beat  Cannot start service superset-worker-beat: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

ERROR: for superset-worker  Cannot start service superset-worker: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown

ERROR: for superset  Cannot start service superset: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "/app/docker/docker-bootstrap.sh": permission denied: unknown
ERROR: Encountered errors while bringing up the project.

### Expected results
All containers to be succesfully deployed by Docker-Compose.

### Actual results
Superset containers failed to be deployed because of error "/app/docker/docker-bootstrap.sh": permission denied: unknown

### How to reproduce the bug
cd incubator-superset-master
docker-compose up

**Comment by SynapseAria** [comment]
same issue

**Comment by waqasakramef** [comment]
same issue

**Comment by aixushuai** [comment]
chmod +x docker/*.sh
![image](https://user-images.githubusercontent.com/8036671/212267891-934cf9d6-55fa-4faf-b822-dd181ac0a11b.png)

**Reply by wyc990122** [reply]
I have run this command,but the question still exists. Here is the log,thanks:
```
[root@wycCentos 0.999dev]# chmod +x docker/*.sh
[root@wycCentos 0.999dev]# docker-compose -f docker-compose.yml up
```
```
superset_tests_worker  | /usr/bin/docker-entrypoint.sh: line 21: /app/docker/docker-bootstrap.sh: Permission denied
superset_tests_worker exited with code 126
superset_worker        | /usr/bin/docker-entrypoint.sh: line 21: /app/docker/docker-bootstrap.sh: Permission denied
superset_init          | /usr/bin/docker-entrypoint.sh: line 21: /app/docker/docker-init.sh: Permission denied
superset_node          | internal/modules/cjs/loader.js:905
superset_node          |   throw err;
superset_node          |   ^
superset_node          | 
superset_node          | Error: Cannot find module '/app/docker/docker-frontend.sh'
superset_node          |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
superset_node          |     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
superset_node          |     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
superset_node          |     at internal/main/run_main_module.js:17:47 {
superset_node          |   code: 'MODULE_NOT_FOUND',
superset_node          |   requireStack: []
superset_node          | }
superset_worker_beat   | /usr/bin/docker-entrypoint.sh: line 21: /app/docker/docker-bootstrap.sh: Permission denied
superset_app           | /usr/bin/docker-entrypoint.sh: line 21: /app/docker/docker-bootstrap.sh: Permission denied
superset_worker exited with code 126
superset_init exited with code 126
superset_node exited with code 1
superset_worker_beat exited with code 126
superset_websocket     | Error: EACCES: permission denied, scandir '/home/superset-websocket'
superset_app exited with code 126
superset_websocket exited with code 0
superset_worker exited with code 126
superset_worker_beat exited with code 126
superset_app exited with code 126
superset_worker exited with code 126
superset_worker_beat exited with code 126
superset_app exited with code 126
superset_worker exited with code 126
superset_worker_beat exited with code 126
superset_app exited with code 126
superset_worker exited with code 126
superset_worker_beat exited with code 126
superset_app exited with code 126
superset_worker exited with code 126
superset_worker_beat exited with code 126
superset_app exited with code 126
superset_worker        | /usr/bin/docker-entrypoint.sh: line 21: /app/docker/docker-bootstrap.sh: Permission denied

```

**Reply by aixushuai** [reply]
Maybe you Should Check your script privilege or docker-compose.yml file
